Presentation.Web/Models/External/V2/ItInterfaceRequestDTO.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace Presentation.Web.Models.External.V2
+{
+    public class ItInterfaceRequestDTO
+    {
+        /// <summary>
+        /// UUID for IT-Interface
```

Comment:
Hvis ikke vi skal lave helt nyt autorisationslag så er vi nødt til at have uuid på den organisation de forskellige entities skal oprettes i, for brugere kan være i 100 organisationer så vi ved ikke umiddelbart i hvilken kontekst de ønsker at arbejde ud fra den bruger der er autoriseret. Det er samme problematik i de andre requests og det er KUN i POST øjemed at det er et problem - i PUT og DELETE er det ikke et problem. I GET vil vi være nødt til at aflevere alt hvad deres bruger giver lov til og det vil sige at hvis brugeren har rettighedshaveradgang i flere organisationer, så vil de få alle de entities som de har lov til  (hvis de spørger på rettighedshaver APIet)

Reply:
Spørgsmålet er om "kontekst" giver mening for disse eksterne API'er? Man kan jo også se det som om, hvis du er en del af 100 organisationer (det er godt nok mange), så bliver dit resultat summen .. leverandører og rettighedshavere er vel typisk ikke en del af alle organisationer og hvis de er, så er det vel fordi de enkelte kommuner betragter dem som lige parter?

Reply:
vi aftaler

- 1 ekstra endpoint hvor brugeren kan GET en liste af navn/uuid for de orgs han/hun er oprettet i
- ifm. POST, skal client stille med uuid på den organisation entity skal oprettes i